index.html

```diff
@@ -89,7 +90,10 @@
         <div class="deck column-full">
           <div class="my-deck">Name of deck</div>
           <div class="deck-image-wrapper">
-            <div class="deck-image">No Cards in Deck</div>
+            <div class="deck-image">
+              <img class="deck-image-cover" src="">
```

Comment:
Needs an `alt` attribute! Since you're using this as a placeholder, just do `alt=""`